examples/customresourceinterpreter/README.md

```diff
@@ -169,6 +169,8 @@ spec:
   externalName: karmada-interpreter-webhook-example.karmada-system.svc.cluster.local
 ```
 
+Or you can do nothing with the target Service, it will fall back to standard Kubernetes service DNS name format: `https://karmada-interpreter-webhook-example.karmada-system.svc:443/interpreter-workload`.
```

Comment:
The word `do nothing` is ambiguous, leave the `Service` unset?

Reply:
> leave the Service unset

That's what it means. I'll revise it.